library/include/G4YieldDataReader.hh

```diff
@@ -0,0 +1,33 @@
+/***********************************************************************\
+ * This software is licensed under the terms of the GNU General Public *
+ * License version 3 or later. See G4CMP/LICENSE for the full license. *
+\***********************************************************************/
+
+/// \file library/include/G4YieldDataReader.hh
+/// \Reads the yield data for Sarkis model from /CrystalMaps/NIEL/SarkisSiIonYield.txt, populates the relevant data and store it in a other file to be used in G4CMPSarkisNIEL. This could be extended for other models that should be implemented the same way. 
+///
+// 20230814  David Sadek
+
+
+#ifndef G4YIELDDATAREADER_HH
+#define G4YIELDDATAREADER_HH
+
+#include "G4String.hh"
+#include "G4PhysicsLogVector.hh"
+#include "globals.hh"
+#include <vector>
+
+class G4YieldDataReader {
```

Comment:
Please rename this to G4CMPYieldDataReader.  Where is the .cc file that implements this function?

Reply:
This was an attempt to create a data reader file for the ionization yield data, but I used a different method. This file is now deleted. Not committed yet.